Lib/test/test_clinic.py

```diff
@@ -1644,11 +1630,6 @@ def test_disallowed_grouping__must_be_position_only(self):
                 [
                 a: object
                 ]
-        """, """
-            with_kwds
-                [
-                **kwds: dict
-                ]
```

Comment:
Could you add also a test for var-positional?
```
            with_varpos
                [
                *args: tuple
                ]
```
If it is an error and has not yet been tested.

If it is an error, and the error message is the same as for keyword arguments here, then it would be reasonable to make the error message for var-keyword the same. Otherwise we can leave a discrepancy.